app/controllers/admin/api/backend_apis_controller.rb

```diff
@@ -48,7 +48,7 @@ def destroy
 
   private
 
-  DEFAULT_PARAMS = %i[name description private_endpoint].freeze
+  DEFAULT_PARAMS = [:name, :description, :private_endpoint, {annotations: {}}].freeze
```

Comment:
IDK how necessary but if we want to be strict :crying_cat_face: 

```suggestion
  DEFAULT_PARAMS = [:name, :description, :private_endpoint, {annotations: {}.freeze}.freeze].freeze
```